python/lsst/ts/m2gui/controltab/tab_utility_view.py

```diff
@@ -117,19 +124,29 @@ def _update_indicator_color(self, indicator, is_fault):
         indicator.setPalette(palette)
 
     @asyncSlot()
```

Comment:
I always have the problem that we need to put the data type in the decorator of `@Slot()` or `@asyncSlot()` or not. For the implementation of `@asyncSlot()`, it just passes all the arguments:
https://github.com/CabbageDevelopment/qasync/blob/master/qasync/__init__.py#L775-L794

Therefore, the question comes back to `@Slot()` itself. Based on the following document:
https://www.pythonguis.com/faq/should-i-decorate-slots-in-pyside2-and-if-so-how/

It said: "As to whether you should decorate the slots, it's a little trickier to answer -- but generally speaking no, you don't need to". It means we do not need the decorator itself as well. The only exception is the use of thread. But since I do not use the thread, I guess I do not even need this.

I will keep my implementation at this moment. If we found the data type is required, I can create a new ticket to go through all the signals in this GUI code to add the data types.

Thanks!